### PR TITLE
Renamed SES email-contact-list code example to send-email; added readme

### PR DESCRIPTION
**Pull request by Doug-AWS**
## Motivation and Context
The code example name was confusing to some users.
We had no readme.

## Description
Renamed the code example and added a readme.

## Testing
There were no changes to the code.

## Checklist
- [ ] I have updated `CHANGELOG.md`
- [ ] I have updated `aws/SDK_CHANGELOG.md` if applicable

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: aws/sdk/examples/ses/README.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+- _REGION_ is the Region in which the client is created.
+  If not supplied, uses the value of the __AWS_REGION__ environment variable.
+  If the environment variable is not set, defaults to __us-west-2__.
+- __-v__ displays additional information.  
+
+### list-contact
+
+This example lists the email addresses of the contacts in a contact list in the Region.
+
+`cargo run --bin ???-??? -- -c CONTACT-LIST [-r REGION] [-v]`
```
Comment (jdisanti) [inline aws/sdk/examples/ses/README.md:67]:
```suggestion
`cargo run --bin list-contacts -- -c CONTACT-LIST [-r REGION] [-v]`
```

**Review by jdisanti — COMMENTED** [review]
Could you also update the SDK changelog?

File: aws/sdk/examples/ses/README.md (hunk 1 above)
Comment (Doug-AWS) [reply aws/sdk/examples/ses/README.md:67]: Oops, fixed.

**Comment by Doug-AWS** [comment]
Change log updated.

File: aws/sdk/examples/ses/README.md (hunk 1 above)
Comment (Doug-AWS) [reply aws/sdk/examples/ses/README.md]: Fixed.

**Comment by jdisanti** [comment]
The changelog entries need to go under `vNext` rather than under the previous release.

**Comment by jdisanti** [comment]
This was already merged into aws-doc-sdk-examples, and since the examples moved there, it is no longer needed in smithy-rs.

**Pull request closed without merging**